libraries/SITL/SITL.cpp

```diff
@@ -168,10 +178,25 @@ const AP_Param::GroupInfo SIM::var_info[] = {
     // @Description: Opflow data delay
     // @Units: ms
     AP_GROUPINFO("FLOW_DELAY",    36, SIM,  flow_delay, 0),
+    // @Param: ADSB_COUNT
+    // @DisplayName: Number of ADSB aircrafts
+    // @Description: Total number of ADSB simulated aircrafts
     AP_GROUPINFO("ADSB_COUNT",    45, SIM,  adsb_plane_count, -1),
+    // @Param: ADSB_RADIUS
+    // @DisplayName: ADSB radius stddev of another aircraft
```

Comment:
Is this right?  I thought this was just a maximum?!

Reply:
I can see the usage: `Aircraft::rand_normal(0, _sitl->adsb_radius_m);`
where `Aircraft::rand_normal` is defined as `double Aircraft::rand_normal(double mean, double stddev)`